### ISSUE TITLE
Feature Request: Allow pm2-meteor.js instead of JSON file for run time environment variables

### ISSUE THREAD

**Issue by todda00** [andruschka/pm2-meteor#61]
I am looking to run pm2-meteor from a CI pipeline and include environment variables from the pipeline, so the sensitive keys are not committed to the repository. Any way the pm2-meteor.json file could be generated from a JS file, so I can include these variables, or alternatively provide environment variables from a command line argument when running `pm2-meteor deploy` ? 

I see that pm2 configuration files allow for Javascript format as well as JSON and YAML, not sure how difficult it would be to allow for a .js config file here.

**Comment by andruschka** [comment]
Yop good idea.

**Comment by andruschka** [comment]
Yes this works now.
pm2-meteor.js :
```
module.exports = {
  appName: process.env.APP_NAME,
  ...
}
```

**Comment by todda00** [comment]
This is great! However, I need to use environment variables in either env.METEOR_SETTINGS, or be able to use a settings.js file for the meteor settings.

I attempted to add the METEOR_SETTINGS directly in the pm2-meteor.js, but its not working. I noticed this is being set to a blank object when there is no settings.json file. https://github.com/andruschka/pm2-meteor/blob/master/src/localTasks.coffee#L76
